src/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "@EXTENSION_NAME@",
   "description": "Urgent post-release fixes for web compatibility.",
-  "version": "101.7.0",
```

Comment:
This should change to 101.8.0 as I'm landing a patch against SmartBlock right now. Or, you could backport that patch as part of this set, if you'd rather do that (https://phabricator.services.mozilla.com/D147309)